--- minitube-orig/src/MediaView.h	2009-05-12 11:45:13.000000000 +0200
+++ minitube/src/MediaView.h	2009-05-28 14:47:43.761187389 +0200
@@ -3,9 +3,40 @@
 
 #include <QtGui>
 #include <QtNetwork>
-#include <phonon>
 #include "View.h"
 #include "ListModel.h"
+#include <phonon/abstractaudiooutput.h>
+#include <phonon/abstractmediastream.h>
+#include <phonon/abstractvideooutput.h>
+#include <phonon/addoninterface.h>
+#include <phonon/audiooutput.h>
+#include <phonon/audiooutputinterface.h>
+#include <phonon/backendcapabilities.h>
+#include <phonon/backendinterface.h>
+#include <phonon/effect.h>
+#include <phonon/effectinterface.h>
+#include <phonon/effectparameter.h>
+#include <phonon/effectwidget.h>
+#include <phonon/mediacontroller.h>
+#include <phonon/medianode.h>
+#include <phonon/mediaobject.h>
+#include <phonon/mediaobjectinterface.h>
+#include <phonon/mediasource.h>
+#include <phonon/objectdescription.h>
+#include <phonon/objectdescriptionmodel.h>
+#include <phonon/path.h>
+#include <phonon/phonondefs.h>
+#include <phonon/phonon_export.h>
+#include <phonon/phononnamespace.h>
+#include <phonon/platformplugin.h>
+#include <phonon/seekslider.h>
+#include <phonon/streaminterface.h>
+#include <phonon/videoplayer.h>
+#include <phonon/videowidget.h>
+#include <phonon/videowidgetinterface.h>
+#include <phonon/volumefadereffect.h>
+#include <phonon/volumefaderinterface.h>
+#include <phonon/volumeslider.h>
 
 class MediaView : public QWidget, public View {
     Q_OBJECT
